block-chain/src/main/java/itx/blockchain/api/MissingReaderException.java

```diff
@@ -0,0 +1,4 @@
+package itx.blockchain.api;
+
+public class MissingReaderException extends Exception {
+}
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding constructors for exception details</b></div>
<div id="fix">

Consider adding a constructor to handle exception message and cause. This would allow better error handling and debugging by providing more context about why the reader is missing.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
 package itx.blockchain.api;

 public class MissingReaderException extends Exception {
    public MissingReaderException(String message) {
        super(message);
    }

    public MissingReaderException(String message, Throwable cause) {
        super(message, cause);
    }
 }
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rkj6124/bito-test/pull/4#issuecomment-2557450846>#0dafab</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged